pkg/registry/apis/folders/folder_storage.go

```diff
@@ -77,6 +77,10 @@ func (s *folderStorage) Create(ctx context.Context,
 	createValidation rest.ValidateObjectFunc,
 	options *metav1.CreateOptions,
 ) (runtime.Object, error) {
+	if 1 == 1 {
```

Comment:
Needed ?

Reply:
not at all, that was inserted to trigger CI checks. Removing it now